### PR TITLE
XSD Schema update

### PR DESCRIPTION
**Pull request by mbriskar**
classification - description is now element
link/xslt - description renamed to title

### CODE REVIEW

**Comment by mbriskar** [comment]
this PR waits until the rules reviewing is done

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/836/<h2>Failed Tests: <span class='status-failure'>5</span></h2><h3><a name='org.jboss.windup.reporting:windup-reporting-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.reporting$windup-reporting-tests/testReport'>org.jboss.windup.reporting:windup-reporting-tests</a>: <span class='status-failure'>2</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.reporting$windup-reporting-tests/testReport/org.jboss.windup.reporting.handlers/ClassificationHandlerTest/testClassificationParsing/'><strong>org.jboss.windup.reporting.handlers.ClassificationHandlerTest.testClassificationParsing</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.reporting$windup-reporting-tests/testReport/org.jboss.windup.reporting.handlers/HintHandlerTest/testHintHandler/'><strong>org.jboss.windup.reporting.handlers.HintHandlerTest.testHintHandler</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-java-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-java-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/JavaClassXmlRulesTest/testJavaClassCondition/'><strong>org.jboss.windup.rules.java.JavaClassXmlRulesTest.testJavaClassCondition</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XMLTransformationXMLRulesTest/testXSLTTransformation/'><strong>org.jboss.windup.rules.xml.XMLTransformationXMLRulesTest.testXSLTTransformation</strong></a></li></ul><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/836/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSourceModeTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSourceModeTest.testRunWindupSourceMode</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/204/
Test FAILed.

File: reporting/impl/src/main/java/org/jboss/windup/reporting/xml/ClassificationHandler.java [hunk 1]
```diff
@@ -70,9 +69,11 @@ public Classification processElement(ParserContext handlerManager, Element eleme
             classification.withSeverity(severity);
         }
 
-        if (StringUtils.isNotBlank(description))
+        Element descriptionChildren = $(element).children("descriptiion").get(0);
```
Comment (lincolnthree) [inline reporting/impl/src/main/java/org/jboss/windup/reporting/xml/ClassificationHandler.java]: descriptiion ;)

**Pull request merged**